File: TestKLine/TestKLine/ViewController.m
//
//  ViewController.m
//  TestKLine
//
//  Created by LRF on 15/9/15.
//  Copyright (c) 2015年 LRF. All rights reserved.
//

#import "ViewController.h"
#import "ImMarketPortraitVC.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}
- (IBAction)onClickKLine:(id)sender {
    ImMarketPortraitVC* klineContentVC = [[ImMarketPortraitVC alloc] initWithNibName:@"ImMarketPortraitVC" bundle:nil];
    
//    TradeCodeData* data = [self getTradeCodeDataWithSection:_controllerDropDownList.nSection withRow:_controllerDropDownList.nRow];
//    klineContentVC.tradeCodeData = data;
//    
    [self presentViewController:klineContentVC animated:YES completion:^()
     {
         
     }];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: FirstAPP/FirstAPP/ViewController.swift
//
//  ViewController.swift
//  FirstAPP
//
//  Created by LRF on 16/1/5.
//  Copyright © 2016年 LRF. All rights reserved.
//

import UIKit

import FirstMixed

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        
        var swiftView = SwiftView(frame: CGRectMake(0, 0, 50, 50))
        self.view .addSubview(swiftView)
        
        var mixedView = MixedView(frame: CGRectMake(50, 0, 50, 50))
        self.view.addSubview(mixedView)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

